### ISSUE TITLE
Connect - Notification Jellyfin broken since update

### ISSUE THREAD

**Issue by bonswouar** [Radarr/Radarr#10026]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

When the notification to Jellyfin is sent an error is returned (see log attached), and the status becomes "Notifications unavailable due to failures: Jellyfin"

### Expected Behavior

Notifications to Jellyfin should work

### Steps To Reproduce

1. Update Jellyfin server to 10.9.0 or 10.9.1
2. Send a notification from Radarr to Jellyfin (for example when a movie is added)
3. See status/error _(side-note: it seems when using the "Test" button from "Edit Notification - Emby / Jellyfin", the 404 isn't considered as an error, the checkmark is green but the error is still in the logs as a warning)_

### Environment

```markdown
- OS: Debian 12
- Radarr: 5.5.3.8819
- Docker Install: Yes
- Using Reverse Proxy: Yes
- Database: Sqlite 3.44.2
```

### What branch are you running?

Master

### Trace Logs? **Not Optional**

```
2024-05-15 12:05:56.2|Warn|HttpClient|HTTP Error - Res: HTTP/1.1 [POST] http://jellyfin_host:8096/mediabrowser/Notifications/Admin: 404.NotFound (0 bytes)
2024-05-15 12:05:56.2|Warn|NotificationService|Unable to send OnDownload notification to: Jellyfin

[v5.5.3.8819] NzbDrone.Common.Http.HttpException: HTTP request failed: [404:NotFound] [POST] at [http://jellyfin_host:8096/mediabrowser/Notifications/Admin]
   at NzbDrone.Common.Http.HttpClient.ExecuteAsync(HttpRequest request) in ./Radarr.Common/Http/HttpClient.cs:line 119
   at NzbDrone.Common.Http.HttpClient.Post(HttpRequest request) in ./Radarr.Common/Http/HttpClient.cs:line 363
   at NzbDrone.Core.Notifications.Emby.MediaBrowserProxy.ProcessRequest(HttpRequest request, MediaBrowserSettings settings) in ./Radarr.Core/Notifications/MediaBrowser/MediaBrowserProxy.cs:line 59
   at NzbDrone.Core.Notifications.Emby.MediaBrowserProxy.Notify(MediaBrowserSettings settings, String title, String message) in ./Radarr.Core/Notifications/MediaBrowser/MediaBrowserProxy.cs:line 31
   at NzbDrone.Core.Notifications.Emby.MediaBrowserService.Notify(MediaBrowserSettings settings, String title, String message) in ./Radarr.Core/Notifications/MediaBrowser/MediaBrowserService.cs:line 34
   at NzbDrone.Core.Notifications.Emby.MediaBrowser.OnDownload(DownloadMessage message) in ./Radarr.Core/Notifications/MediaBrowser/MediaBrowser.cs:line 33
   at NzbDrone.Core.Notifications.NotificationService.Handle(MovieFileImportedEvent message) in ./Radarr.Core/Notifications/NotificationService.cs:line 157
```

### Trace Logs have been provided as applicable. Reports will be closed if the required logs are not provided.

- [X] I have read and followed the steps in the wiki link above and provided the required trace logs - the logs contain `trace` - that are relevant and show this issue.

**Comment by tordenflesk** [comment]
I'm experiencing the same issue with Sonarr as well.

**Comment by bonswouar** [comment]
Related to https://github.com/jellyfin/jellyfin/issues/1681

**Comment by Theetjuh** [comment]
Same issue here, just noticed the error.

**Comment by ddalben** [comment]
Also seeing it from both Sonarr and Radarr

**Comment by gaalma** [comment]
also experiencing the described behaviour in both Sonarr and Radarr

**Comment by jstewart612** [comment]
Did some digging.

I think it happened here:

- https://github.com/jellyfin/jellyfin/pull/8952
- https://github.com/jellyfin/jellyfin/pull/9147

Based on: https://github.com/Sonarr/Sonarr/issues/5828

The closing, it seems, was without proper resolution.  Hence, why we are here now:  because 9147 seems to have hit on 10.9.3:

![image](https://github.com/Radarr/Radarr/assets/32647598/6c7e5f33-d4a6-47e0-bf5c-6c9f40d0ff6e)

**Comment by yourfriendaaron** [comment]
Also having this issue, connections between Radar and Jellyfin are broken, even though test connection reports working.
`HttpClient: HTTP Error - Res: HTTP/1.1 [POST] http://[localip]/mediabrowser/Notifications/Admin: 404.NotFound`

**Comment by thezoggy** [comment]
sharing from sonarr discord from a few months ago:

> Remove unused notification endpoints [PR #8952], by @nielsvanvelzen
https://github.com/jellyfin/jellyfin/pull/8952

remove that notification trigger since its not going to be valid anymore for your version
will still fallback to doing library update if you use that trigger

TL;DR;  You'll want to uncheck Send Notifications and leave Update Library enabled
